mautic/mautic
# Fixing Import custom email template functionality

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ✅ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #12296  



#### Description:

Custom email template no longer loads or renders in template editor box when a custom email template is imported.



#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a couple of emails (template type)
3. Go to the Contact page
4. Select a contact to send a custom email
5. Click on **Send Email** on top of contact page
6. Select an email template from the "Import from an existing template" drop-down menu at the bottom
7. After selecting an email template, the template will load or render in Send Email editor box above
